desktop/start.js

```diff
@@ -1,23 +1,30 @@
 #!/usr/bin/env node
 const portfinder = require('portfinder');
 const concurrently = require('concurrently');
+require('dotenv').config();
 
 const basePort = 8080;
 
 portfinder.getPortPromise({
     port: basePort,
 }).then((port) => {
+    const devServer = `webpack-dev-server --config config/webpack/webpack.dev.js --port ${port} --env.platform desktop`;
+    const buildMain = 'webpack --config config/webpack/webpack.desktop.js --config-name desktop-main --mode=development';
+
     const processes = [
         {
-            command: `webpack-dev-server --config config/webpack/webpack.dev.js --port ${port} --platform desktop`,
+            command: devServer,
             name: 'Renderer',
             prefixColor: 'red.dim',
-
         },
         {
-            command: `wait-port localhost:${port} && export PORT=${port} && electron desktop/main.js`,
+            command: `${buildMain} && wait-port localhost:${port} && electron desktop/dist/main.js`,
             name: 'Main',
             prefixColor: 'cyan.dim',
+            env: {
+                PORT: port,
+                NODE_ENV: 'development',
+            },
```

Comment:
what's possible is to add a 3rd script like
```js
command: `${buildMain} --watch`,
```

And this would rebuild `desktop/dist/main.js` when it changes, but unfortunately the change wasn't getting picked up and I decide it to leave this for after the PR

Reply:
Cool, like I said above I'd be fine addressing this with a follow-up.